app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -2403,6 +2432,22 @@ class ChatController(args: Bundle) :
         if (CapabilitiesUtil.isAbleToCall(conversationUser)) {
             conversationVoiceCallMenuItem = menu.findItem(R.id.conversation_voice_call)
             conversationVideoMenuItem = menu.findItem(R.id.conversation_video_call)
+
+            if (CapabilitiesUtil.hasSpreedFeatureCapability(conversationUser, "silent-call")) {
```

Comment:
- [ ] check if capability is called "silent-call" 
   - https://github.com/nextcloud/spreed/issues/3950#issuecomment-1135703649